Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.h

```diff
@@ -30,15 +30,30 @@
 #include "PDFDocumentLayout.h"
 #include "PDFPluginBase.h"
 #include <WebCore/GraphicsLayer.h>
+#include <wtf/OptionSet.h>
 
 namespace WebKit {
-
 class WebFrame;
+class WebMouseEvent;
 
 class UnifiedPDFPlugin final : public PDFPluginBase, public WebCore::GraphicsLayerClient {
 public:
     static Ref<UnifiedPDFPlugin> create(WebCore::HTMLPlugInElement&);
 
+    enum class ElementUnderMouse : uint16_t {
```

Comment:
probably this should not be about "mouse" but under a "point"

Reply:
(or just more generic "type" or something)

Reply:
`Element` feels wrong in a web context, since it has a very specific meaning in DOM. Also the "under mouse" part of the name seems wrong; this enum is just a list of types. Maybe `InteractableTypes` or something

Reply:
Going with `PDFElementTypes` for now